backend-api/src/functions/asyncActiveSession/configService/configService.ts

```diff
@@ -21,25 +23,41 @@ export class ConfigService implements IGetConfig<Config> {
         errorCategory: "SERVER_ERROR",
       });
     }
-    if (!env.STS_JWKS_ENDPOINT) {
+    if (!env.AUDIENCE) {
       return errorResult({
-        errorMessage: "No STS_JWKS_ENDPOINT",
+        errorMessage: "No AUDIENCE",
         errorCategory: "SERVER_ERROR",
       });
     }
     try {
-      new URL(env.STS_JWKS_ENDPOINT);
+      new URL(env.AUDIENCE);
     } catch {
       return errorResult({
-        errorMessage: "STS_JWKS_ENDPOINT is not a URL",
+        errorMessage: "AUDIENCE is not a URL",
+        errorCategory: "SERVER_ERROR",
+      });
+    }
+    if (!env.STS_BASE_URL) {
+      return errorResult({
+        errorMessage: "No STS_BASE_URL",
+        errorCategory: "SERVER_ERROR",
+      });
+    }
+    try {
+      new URL(env.STS_BASE_URL);
+    } catch {
+      return errorResult({
+        errorMessage: "STS_BASE_URL is not a URL",
         errorCategory: "SERVER_ERROR",
       });
     }
 
     return successResult({
       ENCRYPTION_KEY_ARN: env.ENCRYPTION_KEY_ARN,
       SESSION_TABLE_NAME: env.SESSION_TABLE_NAME,
-      STS_JWKS_ENDPOINT: env.STS_JWKS_ENDPOINT,
+      STS_BASE_URL: env.STS_BASE_URL,
+      STS_JWKS_ENDPOINT: env.STS_BASE_URL + "/.well-known/jwks.json",
```

Comment:
Not sure if we are ok with this?

Reply:
I'm good with this. Assuming that it _doesn't_ make sense to move the ./well-known/jwks.json string closer to the functionality that gets the public key

Reply:
It does make sense to move it closer - it makes a lot of sense! I will do that!

Reply:
not really - it felt better bringing it out of the template though!

Reply:
I have moved this to the PublicKeyGetter. 
STS_JWKS_ENDPOINT has been deleted.